addons/sale_project/models/project.py

```diff
@@ -471,7 +470,9 @@ class ProjectTask(models.Model):
         copy=True, tracking=True, index='btree_not_null', recursive=True,
         compute='_compute_sale_line', store=True, readonly=False,
         domain="[('company_id', '=', company_id), ('is_service', '=', True), ('order_partner_id', '=?', partner_id), ('is_expense', '=', False), ('state', 'in', ['sale', 'done'])]",
-        help="Sales Order Item to which the time spent on this task will be added, in order to be invoiced to your customer.")
+        help="Sales Order Item to which the time spent on this task will be added in order to be invoiced to your customer."
+             "By default the sales order item set on the project will be selected. In the absence of one, the last prepaid sales order item that has time remaining will be used."
+             "Remove the sales order item in order to make this task non billable. You can also change or remove the sales order item of each timesheet entry individually.")
```

Comment:
No need `\n` here? 🤔

Reply:
@xavierbol , Yes, needed over here, Done all the changes. Thanks